build/ci/wait_demo_pr_checks.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -xe
+set +o pipefail
+
+wait_for_checks_to_complete(){
+  community_operators_branch=$1
+  all_tests_passed=false
+  gh_pr_checks_command=(gh pr checks $community_operators_branch --repo $github_csiblock_community_operators_repository)
+  export repo_pr=`gh pr list --repo $github_csiblock_community_operators_repository | grep $community_operators_branch`
+  if [[ "$repo_pr" == *"$community_operators_branch"* ]]; then
+    sleep 5
+    while [ `"${gh_pr_checks_command[@]}" | grep -i pending | wc -l` -gt 0 ]; do
+      sleep 15
+    done
+    summary_counter=0
+    seconds_to_wait_for_summary_test=20
+    while [ "$test_summary" != *"pending"* ] && [ $summary_counter -lt $seconds_to_wait_for_summary_test ] && [ $all_tests_passed == "false" ]; do
+      sleep 1
+      export test_summary=`"${gh_pr_checks_command[@]}" | grep -i summary`
```

Comment:
do we really need to export here?

Reply:
please remove all unneeded `export`s as well (if there are any)

Reply:
done

Reply:
so why is [this one](https://github.com/IBM/ibm-block-csi-operator/blob/e52e5f95eba5ffc7666e47d9d6c5ea3dcdc0871b/build/ci/community/create_demo_pr.sh#L29) needed?

Reply:
oh I thought I removed it
done

Reply:
are they all necessary [here](https://github.com/IBM/ibm-block-csi-operator/blob/e4b57bfa9a6877b16ab85ecf561f6b34a6794e7b/build/ci/community/prepare_env.sh#L6-L10)?